BREAKING_CHANGES.md

```diff
@@ -60,7 +64,7 @@
 * The `useBranchName` magic string has been removed. Instead use `{BranchName}` for `label`.
 
 * The `BranchPrefixToTrim` configuration property has been removed. `RegularExpression` is now used to capture named groups instead.
-  * Default `RegularExpression` for feature branches is changed from `^features?[/-]` to `^features?[/-](?<BranchName>.+)` to support using `{BranchName}` out-of-the-box
+  * Default `RegularExpression` for feature branches is changed from `^features?[\/-]` to `^features?[\/-](?<BranchName>.+)` to support using `{BranchName}` out-of-the-box
```

Comment:
please revert this, if it changed in the new version, add a new section in the new version instead, we should not change the changelog for already released versions

Reply:
What do you mean? `features?[/-]` is not a valid regular expression. I have fixed the documentation

Reply:
ah, ok then